File: vue-webapp/src/classes/communication/RegisterRequest.ts
export default class RegisterRequest {
    
    private username: string;
    private password: string;
    private passwordRepeated: string;

    constructor(username: string, password: string,
                    passwordRepeated: string) {
        this.username = username;
        this.password = password;
        this.passwordRepeated = passwordRepeated;
    }
}

File: vue-backend-communication/src/main/java/com/vuebackend/communication/RegisterRequest.java
package com.vuebackend.communication;

public class RegisterRequest {

    private String username;
	private String password;
	private String passwordRepeated;

	
	public RegisterRequest() {}

	public RegisterRequest(String username, String password, 
							String passwordRepeated) {
        this.username = username;
		this.password = password;
		this.passwordRepeated = passwordRepeated;
    }

    public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}

	public String getUsername() {
		return username;
	}

	public void setUsername(String username) {
		this.username = username;
	}

	public String getPasswordRepeated() {
		return passwordRepeated;
	}

	public void setPasswordRepeated(String passwordRepeated) {
		this.passwordRepeated = passwordRepeated;
	}

	public boolean passwordsNotEqual() {
		return !this.password.equals(this.passwordRepeated);
	}
}